### ISSUE TITLE
How to API Domain.

### ISSUE THREAD

**Issue by chope1** [lincanbin/Android-Carbon-Forum#9]
Hi, please I've created api domain name but I don't know which code to run in there.
The documentation did not help, please which code should be added in the api folder?

**Comment by lincanbin** [comment]
![qq 20160724163950](https://cloud.githubusercontent.com/assets/5785188/17082821/98ef3118-51bd-11e6-8a38-d871e20179b6.png)

**Comment by chope1** [comment]
Yes, I know the settings. I am just asking if any file should be at the root of the api folder.

**Comment by lincanbin** [comment]
api folder???

**Comment by chope1** [comment]
Yes, after creating the api sub-domain, what should be added there? Or should it be empty?
I've created mine @ http://api.techngr.com
What next?

**Comment by lincanbin** [comment]
CName api domain to gpost.techngr.com.

**Comment by chope1** [comment]
Ok. Thanks.
What about the Mobile subdomain  - eg m.techngr.com?

**Comment by lincanbin** [comment]
Right, just like API.

**Comment by pratikraman-dev** [comment]
hello @lincanbin what to do after creating subdomain for api i'm unable to setup as you told above can you please help me out ?
